src/Shopsys/ShopBundle/Resources/config/services.yml

```diff
@@ -64,6 +64,10 @@ services:
 
     Shopsys\FrameworkBundle\Model\Product\ProductDataFactoryInterface: '@Shopsys\ShopBundle\Model\Product\ProductDataFactory'
 
+    Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface: '@Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainElasticFacade'
+
+    Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportRepository: '@Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportWithFilterRepository'
```

Comment:
:smile_cat: shouldn't be here `alias:` :smile_cat:

Reply:
I't the same thing (see [Symfony docs on Aliasing](https://symfony.com/doc/current/service_container/alias_private.html#aliasing)) and here the shorthand looks more consistent with the rest of the services.